.circleci/config.yml

```diff
@@ -96,17 +85,10 @@ workflows:
               only:
                 - master
                 - dev
-      - tag:
```

Comment:
Why did we remove the tag job?

Reply:
Is that handled by the tagging workflow in the .github/workflows folder?

Reply:
it was changed back in Dec 28, 2022 and looks like the tagging.yml is used instead. https://github.com/etn-ccis/blui-react-auth-shared/blob/dev/.github/workflows/tagging.yml
Do we want to cleanup the circleCI config for items not used now or fly with this?

Reply:
Yep, I'm a goober. I missed the part where we moved it into GH Actions.